### PR TITLE
Create sv_SE.json

### PR DESCRIPTION
**Pull request by eson57**
Lets see how this Swedish translation will work. I think I got it all right, but of course there might be minor mistakes that will have to be adjusted.

### CODE REVIEW

File: locales/sv_SE.json [hunk 1]
```diff
@@ -0,0 +1,998 @@
...
+        "0": "Du är framme vid målet.",
+        "1": "Du är framme vid <DESTINATION>.",
+        "2": "Din destination ligger på <RELATIVE_DIRECTION> sida.",
+        "3": "<DESTINATION> ligger på <RELATIVE_DIRECTION> sida."
+      },
+      "relative_directions": ["vänster", "höger"],
+      "example_phrases": {
+        "0": ["Du är framme vid målet."],
+        "1": ["Du är framme vid 3206 Powelton Avenue."],
+        "2": ["Din destination ligger på vänster sida sida.", "Din destination ligger på höger sida."],
```
Comment (kevinkreiser) [inline locales/sv_SE.json]: `sida sida` is probably a typo right?

**Comment by kevinkreiser** [comment]
i found a few issues with the file and fixed them (as i know a bit of swedish myself). we'll need to add tests for this and add the capitalization code that you mentioned earlier for cardinal direction. tack igen så mycket!

**Comment by eson57** [comment]
Thanks for checking and correcting. 👍

**Comment by kevinkreiser** [comment]
ok so the remaining work for tests and capitalization im going to do in a separate pr. ill make an issue for it. thanks @eson57

**Comment by eson57** [comment]
👍

**Pull request merged**